src/Makefile

```diff
@@ -350,8 +352,8 @@ ifeq "$(shell uname -s)" "Darwin"
 		S := $(C_CXX_FLAGS)
 		C_CXX_FLAGS = $(subst -MMD,,$(S))
 
-		C_CXX_FLAGS += -gfull -arch x86_64
```

Comment:
At the beginning of Makefile CPU_ARCH is set to `x64` based on my uname -m of `x86_64`, later CPU_ARCH is set to `x86` based on CPU_ARCH being `x64`. This means that here it'll attempt to build with arch `x86` which my Monterey doesn't have. This will error out with `clang: error: invalid arch name '-arch x86'`

Reply:
Thank you so much for this @Jertzukka! I hadn't tested it on x86 and thought it could fail. I have now tried it on x86 and fixed all the problems with local builds using brew packages. I would appreciate it if you could confirm it on your end 😃